response_operations_ui/views/social_view_case_details.py

```diff
@@ -0,0 +1,25 @@
+import logging
+from flask import Blueprint, render_template
+from flask_login import login_required
+from structlog import wrap_logger
+from response_operations_ui.controllers import case_controller, sample_controllers
+from response_operations_ui.common.mappers import map_social_case_status
+from response_operations_ui.forms import SearchForm
+
+logger = wrap_logger(logging.getLogger(__name__))
+social_bp = Blueprint('social_bp', __name__,
+                      static_folder='static', template_folder='templates')
+
+
+@social_bp.route('/case/<case_id>', methods=['GET'])
+@login_required
+def view_social_case_details(case_id):
+    form = SearchForm()  # TODO - This needs to be edited and expanded on the work done for 'Find case by postcode'
+
+    social_case = case_controller.get_case_by_id(case_id)
```

Comment:
How are we validating that this is case is indeed a social case? What controls do we have in place to ensure the case retrieved is a social case?

Reply:
That's a good point, however there's currently no design in place to handle a 'case not found' scenario directly in the UI. We're using get_case_by_id: https://github.com/ONSdigital/response-operations-ui/blob/0a3fb6016e21672993821e2a1c9fca79a4cfa2f7/response_operations_ui/controllers/case_controller.py#L13
This should raise an exception if the case is not found by UUID, which is provided directly in the URL or from the post code search result.

Reply:
Currently if you put a case_id for a business into the url the page will render but no data will be displayed as the sample attributes are different

Reply:
@emilioward @liamtoozer Maybe we should only enable this functionality for social cases.

Reply:
It depends on what we want to present back to the user